.gitignore

```diff
@@ -0,0 +1,4 @@
+*.pickle
+*.pyc
+*.html
+data/
```

Comment:
Probably shouldn't list `data/` if you have version-controlled files there.

You have a `.gitignore` in the `data` directory anyway.